### PR TITLE
add jspm to "using babel" page under build systems

### PR DESCRIPTION
**Pull request by codylindley**

### CODE REVIEW

File: _includes/tools/jspm/usage.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "baseURL": "/",
+  "transpiler": "babel",
+  "babelOptions": {
+    "optional": [
+      "runtime"
+    ]
+  }
+});
+```
+
```
Comment (sebmck) [inline _includes/tools/jspm/usage.md]: Excessive newlines

File: _includes/tools/jspm/usage.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+```
+when asked which transpiler you would like to use:
+
+> Which ES6 transpiler would you like to use, Traceur or Babel? [traceur]:
+
+Type Babel and press enter.
+
+Note: **Transpiler**: Change this option at any time with `jspm dl-loader --babel`. Custom transpilation options can also be set through `babelOptions` in the jspm config file (i.e. config.js). 
+
+The JSON code below shows the default [Babel options](https://babeljs.io/docs/usage/options/) set in config.js when selecting Babel as the transpiler from the `$jspm init` command.
```
Comment (sebmck) [inline _includes/tools/jspm/usage.md]: No space after the `$`.

Comment (sebmck) [inline _includes/tools/jspm/usage.md]: JSON code => JSON

Comment (sebmck) [inline _includes/tools/jspm/usage.md]: Wrap `config.js` in backticks please.

File: _includes/tools/jspm/usage.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Create a new project configuration (i.e. config.js):
+
+```sh
+$ jspm init
+```
+when asked which transpiler you would like to use:
+
+> Which ES6 transpiler would you like to use, Traceur or Babel? [traceur]:
+
+Type Babel and press enter.
```
Comment (sebmck) [inline _includes/tools/jspm/usage.md]: Babel => \`babel\` (with backticks) to accurately reflect what users actually type.

File: _includes/tools/jspm/usage.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+Create a new project configuration (i.e. config.js):
+
+```sh
+$ jspm init
+```
```
Comment (sebmck) [inline _includes/tools/jspm/usage.md]: Newline after this code block.

File: _includes/tools/jspm/usage.md (hunk 1 above)
Comment (codylindley) [inline _includes/tools/jspm/usage.md]: Not sure I understand. Can you clarify/specify the change you'd like me to make?

**Comment by guybedford** [comment]
@codylindley thanks for driving this! I'm wondering if jspm installation instructions should be included here at all? Perhaps rather the usage instructions you have of `jspm dl-loader --babel` and `jspm init` and select babel are the "babel install" instructions for jspm only?

Then instead of spelling out the full configuration jspm adds for Babel, just mention something along the lines of that by default jspm will add configuration to use the Babel runtime transformer for modular runtime loading but this can be disabled or other configuration options can be added as well here.

**Comment by codylindley** [comment]
@guybedford - Please change it as you see fit. I'll follow your lead and @sebmck lead.

**Comment by sebmck** [comment]
Closed in favor of #283. Thanks @codylindley!

**Pull request closed without merging**